spec/api-web-contents-spec.js

```diff
@@ -597,19 +597,6 @@ describe('webContents module', () => {
       }
     })
 
-    it('can set the correct zoom level (callback)', async () => {
-      try {
-        await w.loadURL('about:blank')
-        const zoomLevel = await new Promise(resolve => w.webContents.getZoomLevel(resolve))
-        expect(zoomLevel).to.eql(0.0)
-        w.webContents.setZoomLevel(0.5)
-        const newZoomLevel = await new Promise(resolve => w.webContents.getZoomLevel(resolve))
-        expect(newZoomLevel).to.eql(0.5)
-      } finally {
-        w.webContents.setZoomLevel(0)
-      }
-    })
```

Comment:
DYK what happens if one of these calls throws an exception and there's not a `catch` block here? I expect that mocha/chai would catch it itself and fail the test -- or the entire thing would just die, which would still show up in CI as a failed test -- but do we know it for sure?